testing/citest/spinnaker_testing/spinnaker_test_scenario.py

```diff
@@ -423,18 +466,19 @@ def __init__(self, bindings, agent=None):
       agent: [SpinnakerAgent] The Spinnaker agent to bind to the scenario.
     """
     super(SpinnakerTestScenario, self).__init__(bindings, agent)
-    agent = self.agent
+    agent=self.agent
```

Comment:
Odd that you put spaces in keyword parameters but not in statements.
My understanding of standard python formatting is the opposite.
FYI https://www.python.org/dev/peps/pep-0008/#other-recommendations.

When you sync to head and move your changes, dont include any of this reformatting.